marlowe-dashboard-client/src/Capability/PlutusApps/MarloweApp/Types.purs

```diff
@@ -32,11 +40,24 @@ data LastResult
 
 derive instance genericLastResult :: Generic LastResult _
 
-instance encodeLastResult :: Encode LastResult where
-  encode a = genericEncode defaultOptions a
-
-instance decodeLastResult :: Decode LastResult where
-  decode a = genericDecode defaultOptions a
+instance encodeLastResult :: EncodeJson LastResult where
```

Comment:
Why did you change the way this was encoded? is genericEncode no longer an option?

Reply:
No, it isn't - that was from our patched version of `foreign-generic`. There is a `argonaut-generic` that works in a similar way, but the encoding is not compatible with Aeson. Hence the  explicit encode / decode instances.